### ISSUE TITLE
About data processing

### ISSUE THREAD

**Issue by xuxw98** [jzhzhang/3DAwareNav#5]
Hi,
Thanks for your great work. We are very interested in your online 3d segmentation method. We have tried to evaluate the model provided in https://github.com/jzhzhang/FusionAwareConv on scannet validation set, but find there is no scripts for processing data into the required h5 file. I would really appreciate it if you could share this part of code.
We also send an email to your gmail for some additional questions. Looking forward to your reply. Thanks a lot!

**Comment by jzhzhang** [comment]
Thanks for your interest in our project. I already sent the data and training code to you.
